pkgs/development/tools/misc/gnum4/default.nix

```diff
@@ -7,27 +7,19 @@
 
 stdenv.mkDerivation rec {
   pname = "gnum4";
-  version = "1.4.18";
+  version = "1.4.19";
 
   src = fetchurl {
     url = "mirror://gnu/m4/m4-${version}.tar.bz2";
-    sha256 = "1xkwwq0sgv05cla0g0a01yzhk0wpsn9y40w9kh9miiiv0imxfh36";
+    sha256 = "sha256-swapHA/ZO8QoDPwumMt6s5gf91oYe+oyk4EfRSyJqMg=";
   };
 
   doCheck = false;
 
   configureFlags = [ "--with-syscmd-shell=${stdenv.shell}" ];
 
   # Upstream is aware of it; it may be in the next release.
```

Comment:
Presumably this comment was referring to the patch that you removed?

Reply:
That is right. Fixed.